components/brave_welcome_ui/reducers/welcome_reducer.ts

```diff
@@ -42,7 +42,17 @@ const welcomeReducer: Reducer<Welcome.State | undefined> = (state: Welcome.State
       chrome.send('setDefaultSearchEngine', [modelIndex])
       break
     case types.IMPORT_DEFAULT_SEARCH_PROVIDERS_SUCCESS:
-      state = { ...state, searchProviders: payload }
+      // TODO(bsclifton): remove when ready for other regions
```

Comment:
Should we also restrict Brave Search to the `en` locale? I don't think it's localized yet...

Reply:
Good question - there is only one string which needs localizing (the blurb which shows under `Brave Search beta` on onboarding). Ran by @anthonypkeane: we should be good to go with English only - our next translation push/pull will get that blurb translated 😄